docs/using-the-jdbc-driver/sample-code/HikariSample.java

```diff
@@ -49,7 +50,8 @@ public static void main(String[] args) throws SQLException {
     targetDataSourceProps.setProperty("serverPropertyName", "db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com");
     targetDataSourceProps.setProperty("databasePropertyName", "postgres");
     targetDataSourceProps.setProperty("portPropertyName", "5432");
-    ds.setTargetDataSourceProperties(targetDataSourceProps);
+    
+    ds.addDataSourceProperty("targetDataSourceProps", targetDataSourceProps);
```

Comment:
Property name should be "targetDataSourceProperties"

Reply:
Addressed in https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/213/commits/9518e5c47a38815df720af2edbf56f397be861a3